Lifted from Arch; get rid of gnu triplet in gala plugins path

diff -rupN wingpanel-rewrite-x11.orig/wingpanel-interface/CMakeLists.txt wingpanel-rewrite-x11/wingpanel-interface/CMakeLists.txt
--- wingpanel-rewrite-x11.orig/wingpanel-interface/CMakeLists.txt	2015-06-04 15:30:20.002739439 +0200
+++ wingpanel-rewrite-x11/wingpanel-interface/CMakeLists.txt	2015-06-04 15:31:35.983849788 +0200
@@ -30,4 +30,4 @@ OPTIONS
 add_library (${WINGPANELINTERFACE} MODULE ${VALA_C})
 target_link_libraries(${WINGPANELINTERFACE} ${DEPS_LIBRARIES})
 
-install (TARGETS ${WINGPANELINTERFACE} DESTINATION "${CMAKE_INSTALL_PREFIX}/lib/x86_64-linux-gnu/gala/plugins")
+install (TARGETS ${WINGPANELINTERFACE} DESTINATION "${CMAKE_INSTALL_PREFIX}/lib/gala/plugins")
